patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,17 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Used as a last resort for determining potion effect sort order in the gui.
++     * This can be used to preserve a potion effect's sort order if it has a liquid color that is subject to change.
++     * @param The PotionEffect instance containing the potion
++     * @return An index to use to sort potion effects if the value is different than that of the other effect's potion
```

Comment:
This line is a bit clunky. Maybe just "a value used to sort `PotionEffect`s in GUIs. Defaults to this `Potion`'s color." Also, the same rules for capitalization and punctuation as used for `@param` are used for `@return`. [Reference](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return)